includes/class-newspack-popups-inserter.php

```diff
@@ -535,6 +536,22 @@ public static function enqueue_scripts() {
 			return;
 		}
 
+		if ( Newspack_Popups_Segmentation::is_admin_user() ) {
+			\wp_enqueue_script(
+				'newspack-popups-admin-bar',
+				plugins_url( '../dist/admin.js', __FILE__ ),
+				[],
+				filemtime( dirname( NEWSPACK_POPUPS_PLUGIN_FILE ) . '/dist/admin.js' ),
```

Comment:
I knew that, but in my local testing it unexpectedly worked in all three environments. Turns out it's because I had Transitional mode turned on for testing another feature. 🤦

Reply:
7cb7339 updates the feature so that it's only available in AMP Plus or non-AMP mode, but I'm having trouble getting the styles in `admin.css` to get picked up by AMP...

Reply:
The `newspack-popups-hide-prompts` classname is not in DOM when the site loads, so it gets stripped by AMP. To avoid that, an empty element with this classname can be attached to the DOM.